### PR TITLE
Add Integrated POS Doc

### PR DESCRIPTION
**Pull request by afiebig**
This PR add Documentation and Reference for:

- Open a Serial port
- Close Serial port
- Polling
- Load Keys
- Close Register
- Change to Normal Mode

### CODE REVIEW

File: documentacion/como_empezar/README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ gem install transbank-sdk
 
 Te recomendamos leer [las instrucciones de instalación detalladas para el SDK Ruby](https://github.com/TransbankDevelopers/transbank-sdk-ruby#instalaci%C3%B3n) para mas opciones de instalación.
 
-(Para webpay en Ruby puedes seguir usando [libwebpay](https://github.com/TransbankDevelopers/libwebpay-ruby) u otra alternativa)
+(Para Webpay en Ruby puedes seguir usando [libWebpay](https://github.com/TransbankDevelopers/libwebpay-ruby) u otra alternativa)
```
Comment (leosoto) [inline documentacion/como_empezar/README.md]: Next time try to make these unrelated changes on a different branch/PR

File: documentacion/posintegrado/README.md [hunk 2]
```diff
@@ -0,0 +1,275 @@
+# POS Integrado
+
+## Cómo empezar
+
+El SDK para POS Integrado, cuenta de 2 partes, la primera pieza es una librería/SDK escrito en C,
+el cual es multi plataforma, y debe ser instalado en tu maquina para poder ser utilizado por la
+segunda pieza, la cual mediante un Wrapper, accede a las funciones disponibles en la librería C.
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]: There are a lot of commas there (and a single long sentence!). Maybe you can let the reader take a break somewhere in the middle of that paragraph :)

File: documentacion/posintegrado/README.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+
+Adicionalmente, necesitaras tener instalados los drivers correspondientes a tu tarjeta de
+puerto serial o adaptador USB Serial.
+
+<aside class="notice">
+La comunicación con el POS Integrado se realiza mediante puerto serial RS232.
+</aside>
+
+<aside class="notice">
+Tu eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]:
```suggestion
Tú eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
```

Comment (leosoto) [reply documentacion/posintegrado/README.md]: Maybe we should add some examples or information on what driver is known to work with what adapters?

File: documentacion/posintegrado/README.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+</aside>
+
+### LibSerialPort
+
+El SDK depende de [libSerialPort](https://sigrok.org/wiki/Libserialport) para la comunicación serial.
+
+Puedes obtener el código desde el repositorio usando git:
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]: Do people need the source? Or is it just for advanced usage? (the content under `documentacion` should be kept as straightforward as possible so that people get to use our SDK in the common way as fast as possible; the `reference` section is where we put all the details and extra info)

Comment (leosoto) [reply documentacion/posintegrado/README.md]: Related note: I see there are binary distributions of libserialport: https://sigrok.org/wiki/Downloads#Binaries_and_distribution_packages

File: documentacion/posintegrado/README.md [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+```c
+#include "transbank_serial_utils.h"
+//...
+char * ports = list_ports();
+```
+
+### Abrir un puerto Serial
+
+Para abrir un puerto serial y comunicarte con el POS Integrado, necesitaras el nombre del puerto (El cual
+puedes identificar usando la función mencionada en el apartado anterior #Listar puertos disponibles). También
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]:
```suggestion
puedes identificar usando [la función mencionada en el apartado anterior](#listar-puertos-disponibles). También
```

File: documentacion/posintegrado/README.md [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+necesitaras el baudrate al cual esta configurado el puerto serial del POS Integrado (Por defecto es 115200),
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]: If the default is 115200, I would not add more complexity to the "quickstart"-style documentation. But certainly all this stuff about `TbkBaudrates` is very useful on the reference section.

File: documentacion/posintegrado/README.md [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+Si el puerto no puede ser abierto, se lanzara una exception `TransbankException`.
+
+<div class="language-simple" data-multiple-language></div>
+
+```csharp
+using Transbank.POS;
+using Transbank.POS.Utils;
+//...
+string portName = "COM3";
+POS.Instance.OpenPort(portName, TbkBaudrate.TBK_115200);
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]: Can I just use `POS.Instance.OpenPort(portName)`? (As per the previous comment: If 115200 is the default for the POS integrado, it should be an optional parameter only used by those who have POS integrado with non default values)

File: documentacion/posintegrado/README.md [hunk 8]
```diff
@@ -0,0 +1,275 @@
...
+```
+
+```c
+#include "transbank.h"
+#include "transbank_serial_utils.h"
+//...
+char* portName = "COM4";
+int retval = open_port(portName, 115200);
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]: FWIW, I think in C it's fine to require the baudrate :). I just think that on the wrappers we have to go the extra mile to make the developer experience the best possible

File: documentacion/posintegrado/README.md [hunk 9]
```diff
@@ -0,0 +1,275 @@
...
+```c
+#include "transbank.h"
+#include "transbank_serial_utils.h"
+//...
+char* response = sale(ammount, ticket, false);
+```
+
+### Transacción de Cierre
+
+Este comando es gatillado por la caja y no recibe parámetros. El POS ejecuta la transacción de cierre contra el Autorizador (no se contempla Batch Upload). Como respuesta el POS Integrado enviará un aprobado o rechazado. (Puedes ver la tabla de respuestas en este [link](/referencias/posintegrado#tabla-de-respuestas))
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]: The link should point to "referencia" without the final "s". 

Also: I don't see that content in this PR

File: documentacion/posintegrado/README.md [hunk 10]
```diff
@@ -0,0 +1,275 @@
...
+
+<div class="language-simple" data-multiple-language></div>
+
+```csharp
+using Transbank.POS;
+using Transbank.POS.Responses;
+//...
+RegisterCloseResponse response = POS.Instance.RegisterClose();
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]: Is "RegisterClose" the more natural name for this transaction? Maybe `CloseRegister` or just `Close` might work better? I don't really know, but I recommend giving it a thought before 1.0

File: documentacion/posintegrado/README.md [hunk 11]
```diff
@@ -0,0 +1,275 @@
...
+```json
+"FunctionCode": 510
+"ResponseMessage": "Aprobado"
+"Success": true
+"CommerceCode": 550062700310
+"TerminalId": "ABC1234C"
+```
+
+<aside class="notice">
+Para el cierre no se solicitara tarjeta supervisora.
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]:
```suggestion
Para el cierre no se solicitará tarjeta supervisora.
```

Comment (leosoto) [reply documentacion/posintegrado/README.md]: What does it mean that the supervisor card won't be required? Is that something that is required on the normal POS and changes with the integrated POS?

File: documentacion/posintegrado/README.md [hunk 12]
```diff
@@ -0,0 +1,275 @@
...
+</aside>
+
+### Transacción de Carga de Llaves
+
+Esta transacción permite al POS Integrado del comercio requerir cargar nuevas _Working Keys_ desde Transbank. Como respuesta el POS Integrado enviará un aprobado o rechazado. (Puedes ver la tabla de respuestas en este [link](/referencia/posintegrado#tabla-de-respuestas))
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]: This describes what the transaction does, but not the why/when. When is the integrator supposed to call this thing? Why?

File: documentacion/posintegrado/README.md [hunk 13]
```diff
@@ -0,0 +1,275 @@
...
+
+```c
+#include "transbank.h"
+#include "transbank_serial_utils.h"
+//...
+BaseResponse response = load_keys();
+}
+```
+
+El resultado del cierre de caja se entrega en la forma de un objeto `RegisterCloseResponse`o una estructura `BaseResponse` en el caso de la librería C. Si ocurre algún error al ejecutar la acción en el POS se lanzara una excepción del tipo `TransbankRegisterCloseException`.
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]: Copy/paste error? (We are not on the "cierre de caja" transaction here)

File: documentacion/posintegrado/README.md [hunk 14]
```diff
@@ -0,0 +1,275 @@
...
+"Success": true
+"CommerceCode": 550062700310
+"TerminalId": "ABC1234C"
+```
+
+<aside class="warning">
+El uso de esta transacción debe ser limitado a pruebas de comunicación o cuando el POS Integrado pierda las llaves.
+</aside>
+
+### Transacción de Pooling
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]:
```suggestion
### Transacción de Polling
```

File: documentacion/posintegrado/README.md [hunk 15]
```diff
@@ -0,0 +1,275 @@
...
+
+Esta mensaje es enviado por la caja para saber si el POS está conectado. En el SDK el resultado de esta operación es un `Booleano` o un `0` representado en la constante `TBK_OK` en el caso de la librería en C. Si ocurre algún error al momento de ejecutar la acción en el POS, se lanzara una excepción del tipo `TransbankException`.
+
+<div class="language-simple" data-multiple-language></div>
+
+```csharp
+using Transbank.POS;
+//...
+bool connected = POS.Instance.Polling();
```
Comment (leosoto) [inline documentacion/posintegrado/README.md]: I suggest we call this method `Poll` rather than `Polling`.

File: producto/posintegrado/README.md [hunk 16]
```diff
@@ -0,0 +1,158 @@
+# POS Integrado
+
+El POS integrado, permite realizar transacciones con tarjeta de Crédito/Debito con Transbank utilizando el un puerto serial de un PC o Caja. La comunicación con Transbank y la lógica del procesamiento de una transacción financiera es realizada por el equipo POS, facilitando así la integración con un sistema de caja.
```
Comment (leosoto) [inline producto/posintegrado/README.md]:
```suggestion
El POS integrado permite realizar transacciones con tarjeta de Crédito/Debito con Transbank utilizando el un puerto serial de un PC o Caja. La comunicación con Transbank y la lógica del procesamiento de una transacción financiera es realizada por el equipo POS, facilitando así la integración con un sistema de caja.
```

File: producto/posintegrado/README.md [hunk 17]
```diff
@@ -0,0 +1,158 @@
...
+Ante cualquier cambio que efectúe el Establecimiento, ya sea de Sistema Operativo o Software de Caja, podrían existir impactos en la Integración efectuada con el POS Integrado, por lo que se sugiere probar la correcta operación con POS Integrado antes de efectuar el cambio en ambiente Productivo.
+</aside>
+
+## Seguridad
+
+### Confidencialidad de la Información
+
+De acuerdo a las normativas vigentes, las transacciones con Tarjeta de Crédito y Débito requieren la
+incorporación de los siguientes elementos de seguridad en el sistema:
+La información leída en el punto de venta NO debe ser almacenada en ningún sistema.
```
Comment (leosoto) [inline producto/posintegrado/README.md]: What information does the SDK provide that should not be stored?

Comment (leosoto) [reply producto/posintegrado/README.md]: ...Or more broadly, what information can be read on the responses the POS gives on the serial port that should not be stored

File: producto/posintegrado/README.md [hunk 18]
```diff
@@ -0,0 +1,158 @@
...
+### Tratamiento de los Tracks de la Tarjeta
+
+La información grabada en el Track I y Track II será leída sólo por los dispositivos de seguridad (POS). Estos
+dispositivos procederán a encriptar el contenido del Track I y Track II para que esta información no quede en
+claro.
+
+### Tratamiento de datos sensibles
+
+Para asegurar la confidencialidad de la información, los mensajes de las transacciones o al menos los datos
+sensibles (además del PIN) deben viajar encriptados en los distintos tramos de la conexión, tanto en el
```
Comment (leosoto) [inline producto/posintegrado/README.md]: Isn't that something that the POS does by itself? AFAIU, the integrator doesn't need to anything special for this  to be true. In that case I would change the wording from "deben viajar encriptados" por "viajan encriptados".

File: producto/posintegrado/README.md [hunk 19]
```diff
@@ -0,0 +1,158 @@
...
+requerimiento como en la respuesta. Como datos sensibles se consideran: número de tarjeta, fecha
+expiración, número de cuenta y monto de la transacción (dato validado en Autenticación de Mensajes o MAC).
+
+### El modelo Master/Session KEY
+
+El método actual de administración de llaves es el llamado Máster/Session Key, en el cual los PED (Pin Entry
```
Comment (leosoto) [inline producto/posintegrado/README.md]: Is there a meaningful difference between a "POS" and a "PED" in the context of the POS integrado? If not, I would refrain from introducing more terminology that doesn't serve any purpose from the point of view of the integrator or merchant.

File: producto/posintegrado/README.md [hunk 20]
```diff
@@ -0,0 +1,158 @@
...
+Con la Working Key, se cifra el PinBlock y se envía al servidor.
+
+La Working Key se cambia en forma periódica (al menos en cada cierre), para evitar que sea descubierta por
+terceros.
+
+Este modelo de administración de llaves es el que se usará para las llaves MAC.
+
+### El modelo DUKPT – Encriptación de PIN
+
+El nuevo método de administración de llaves para PIN que usará Transbank es el denominado ―Llave Única
```
Comment (leosoto) [inline producto/posintegrado/README.md]: Usará? Since when? And how that impacts me as an integrator or merchant?

File: producto/posintegrado/README.md [hunk 21]
```diff
@@ -0,0 +1,158 @@
...
+
+Las llaves criptográficas para la generación de MAC (working key de MAC) se maneja de acuerdo a lo siguiente:
+Las working key son generadas por el sistema de Transbank y transmitidas en línea para cada uno de los
+terminal ID definidos en el comercio cliente.
+Para la carga y / o cambio de la working keys de MAC se utilizan las transacciones de CIERRE BATCH y CARGA
+DE LLAVE (Ver Transacciones Administrativas).
+
+Las llaves working keys de MAC se actualizan en cada nueva transacción atendida por Transbank. Por lo que
+la caja debe registrar esta nueva llave para su uso en la siguiente transacción.
+Las llaves se deben cambiar automáticamente todos los días. Esto implica que debe existir un procedimiento
```
Comment (leosoto) [inline producto/posintegrado/README.md]: Oh, so the "Carga de Llaves" transaction should be done daily by the integrator using the SDK?

File: producto/posintegrado/README.md [hunk 22]
```diff
@@ -0,0 +1,158 @@
...
+de inicialización o cierre obligatorio en cada caja (terminal ID) que se ejecuta en forma automática todos los
+días y que como parte de este procedimiento se envía a Transbank una transacción de CIERRE BATCH o CARGA
+DE LLAVE por cada caja (terminal ID).
```
Comment (leosoto) [inline producto/posintegrado/README.md]: Wait, so if I do a "Cierre" I don't need a "Carga de Llaves" because the "Cierre" also reloads the keys?

File: documentacion/posintegrado/README.md (hunk 4 above)
Comment (afiebig) [reply documentacion/posintegrado/README.md]: The binaries for libSerialport are only available for Linux. I already tried the windows packages and they are only for the other apps from Sigrock (pulseview and sigrok-cli) which doesn't include libSerialport . 

Maybe we could compile and upload libSerialport dll alongside the Wrapper.

File: documentacion/posintegrado/README.md (hunk 9 above)
Comment (afiebig) [reply documentacion/posintegrado/README.md]: It is [here](https://github.com/TransbankDevelopers/transbank-developers-docs/pull/75/files/feb80d3523e94b7fd9d1ec891c4d9faed6f3ae76#diff-beff8b3c5e1f8f61a3e3ce572380a1fd) github hides it because it's a large file.

File: documentacion/posintegrado/README.md (hunk 11 above)
Comment (afiebig) [reply documentacion/posintegrado/README.md]: I think it should say `supervisor password` but i will need to confirm this with Carlos. In normal mode, the POS requires the supervisor password to perform some actions.

File: documentacion/posintegrado/README.md (hunk 10 above)
Comment (afiebig) [reply documentacion/posintegrado/README.md]: I will change it to Close, i doubted at first, because it can be confused with the port close action.

File: producto/posintegrado/README.md (hunk 17 above)
Comment (afiebig) [reply producto/posintegrado/README.md]: For what i know, you can't access the readed information, only what the POS provides as responses of a message.

This section is copy/paste from the "MANUAL POS INTEGRADO version 3.0" pdf

File: producto/posintegrado/README.md (hunk 19 above)
Comment (afiebig) [reply producto/posintegrado/README.md]: Have to check this with Carlos. I assumed it was talking about the external keyboard you could connect to the pos in order to get the user PIN.

File: producto/posintegrado/README.md (hunk 22 above)
Comment (afiebig) [reply producto/posintegrado/README.md]: Yes, and it also says that with every transaction processed by Transbank it reloads the keys.

Maybe it's a good idea to include the LoadKeys in the initialization after Serial Port is connected.

File: documentacion/posintegrado/README.md [hunk 23]
```diff
@@ -213,10 +213,10 @@ BaseResponse response = load_keys();
 }
 ```
 
-El resultado del cierre de caja se entrega en la forma de un objeto `CloseResponse`o una estructura `BaseResponse` en el caso de la librería C. Si ocurre algún error al ejecutar la acción en el POS se lanzara una excepción del tipo `TransbankCloseException`.
+El resultado de la carga de llaves entrega en la forma de un objeto `LoadKeysResponse`o una estructura `BaseResponse` en el caso de la librería C. Si ocurre algún error al ejecutar la acción en el POS se lanzara una excepción del tipo `TransbankLoadKeysException`.
```
Comment (leonelmunozlopez) [inline documentacion/posintegrado/README.md]: typo: lanzara -> lanzará

File: documentacion/posintegrado/README.md [hunk 24]
```diff
@@ -213,10 +213,10 @@ BaseResponse response = load_keys();
...
 
 ```json
-"FunctionCode": 510
+"FunctionCode": 810
```
Comment (leonelmunozlopez) [inline documentacion/posintegrado/README.md]: We can use a standard formatted JSON
```json
{
  "FunctionCode": 810,
  "ResponseMessage": "Aprobado",
  ...
}
```

File: producto/posintegrado/README.md [hunk 25]
```diff
@@ -4,15 +4,15 @@ El POS integrado permite realizar transacciones con tarjeta de Crédito/Debito c
...
-- Ultima Venta.
-- Detalle de Ventas.
-- Totales.
-- Poll.
-- Cambio a modalidad POS Normal.
+- [x] Carga de Llaves.
+- [x] Venta.
+- [x] Cierre.
+- [ ] Anulación.
+- [ ] Ultima Venta.
```
Comment (leonelmunozlopez) [inline producto/posintegrado/README.md]: typo: Última

File: producto/posintegrado/README.md [hunk 26]
```diff
@@ -54,105 +54,69 @@ Ante cualquier cambio que efectúe el Establecimiento, ya sea de Sistema Operati
...
 ### Confidencialidad de la Información
 
-De acuerdo a las normativas vigentes, las transacciones con Tarjeta de Crédito y Débito requieren la
-incorporación de los siguientes elementos de seguridad en el sistema:
-La información leída en el punto de venta NO debe ser almacenada en ningún sistema.
-Para efectos de cuadratura e identificación de transacciones, se debe utilizar el NÚMERO DE OPERACION.
+De acuerdo a las normativas vigentes, las transacciones con Tarjeta de Crédito y Débito incorporan los siguientes elementos de seguridad en el sistema:
+
+- La información leída en el punto de venta NO es almacenada en ningún sistema.
+- Para efectos de cuadratura e identificación de transacciones, se utilizar el NÚMERO DE OPERACION.
```
Comment (leonelmunozlopez) [inline producto/posintegrado/README.md]: ... se utiliza el NÚMERO DE OPERACIÓN ...

File: documentacion/posintegrado/README.md (hunk 24 above)
Comment (afiebig) [reply documentacion/posintegrado/README.md]: This is not a Json response, I just used a json block

File: documentacion/como_empezar/README.md [hunk 27]
```diff
@@ -73,7 +73,7 @@ pip install transbank-sdk
 
 Te recomendamos leer [las instrucciones de instalación detalladas para el SDK Python](https://github.com/TransbankDevelopers/transbank-sdk-python#instalaci%C3%B3n) para mas opciones de instalación.
 
-(Para webpay en Python puedes seguir usando [libwebpay](https://github.com/TransbankDevelopers/libwebpay-python), pero te recomendamos usar [python-tbk, creada por Cornershop](https://github.com/cornershop/python-tbk) que será la base de lo que integremos finalmente en transbank-sdk)
+(Para Webpay en Python puedes seguir usando [libWebpay](https://github.com/TransbankDevelopers/libwebpay-python), pero te recomendamos usar [python-tbk, creada por Cornershop](https://github.com/cornershop/python-tbk) que será la base de lo que integremos finalmente en transbank-sdk)
```
Comment (leosoto) [inline documentacion/como_empezar/README.md]: I don't think that `libWebpay` is right. Pretty sure it's called `libwebpay` all lowercase

File: documentacion/posintegrado/README.md (hunk 4 above)
Comment (leosoto) [reply documentacion/posintegrado/README.md]: Fair enough. We can leave that later for the support/evolution of the product and go live with the "use the source, luke" method :)

File: documentacion/posintegrado/README.md (hunk 12 above)
Comment (leosoto) [reply documentacion/posintegrado/README.md]: @afiebig I think this is still pending

File: referencia/webpay/README.md [hunk 28]
```diff
@@ -1468,7 +1468,7 @@ var result = transaction.reverseTransaction(buyOrderLong);
 
 Nombre  <br> <i> tipo </i> | Descripción
 ------   | -----------
-buyorder  <br> <i> xs:long </i> | Orden de compra de la transacción a reversar.
+buyOrder  <br> <i> xs:long </i> | Orden de compra de la transacción a reversar.
```
Comment (leosoto) [inline referencia/webpay/README.md]: Nice catch :)

File: documentacion/como_empezar/README.md [hunk 29]
```diff
@@ -62,7 +62,7 @@ gem install transbank-sdk
 
 Te recomendamos leer [las instrucciones de instalación detalladas para el SDK Ruby](https://github.com/TransbankDevelopers/transbank-sdk-ruby#instalaci%C3%B3n) para mas opciones de instalación.
```
Comment (leonelmunozlopez) [inline documentacion/como_empezar/README.md]: ...para más opciones...

File: documentacion/como_empezar/README.md [hunk 30]
```diff
@@ -62,7 +62,7 @@ gem install transbank-sdk
...
 
-(Para webpay en Ruby puedes seguir usando [libwebpay](https://github.com/TransbankDevelopers/libwebpay-ruby) u otra alternativa)
+(Para Webpay en Ruby puedes seguir usando [libwebpay](https://github.com/TransbankDevelopers/libwebpay-ruby) u otra alternativa)
 
 [En
```
Comment (leonelmunozlopez) [inline documentacion/como_empezar/README.md]: En [**Python**](link)...

File: documentacion/posintegrado/README.md [hunk 31]
```diff
@@ -0,0 +1,305 @@
+# POS Integrado
+
+## Cómo empezar
+
+El SDK para POS Integrado cuenta de 2 partes, una librería/SDK en C, que debe ser instalado en tu maquina para poder ser utilizado por la segunda pieza. Esta pieza es un Wrapper para accede a las funciones disponibles en la librería C.
```
Comment (leonelmunozlopez) [inline documentacion/posintegrado/README.md]: máquina

File: documentacion/posintegrado/README.md [hunk 32]
```diff
@@ -0,0 +1,305 @@
...
+
+Adicionalmente, necesitaras tener instalados los drivers correspondientes a tu tarjeta de
```
Comment (leonelmunozlopez) [inline documentacion/posintegrado/README.md]: necesitarás

File: documentacion/posintegrado/README.md [hunk 33]
```diff
@@ -0,0 +1,305 @@
...
+La comunicación con el POS Integrado se realiza mediante puerto serial RS232.
+</aside>
+
+<aside class="notice">
+Tú eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
+</aside>
+
+<aside class="success">
+Estos drivers son conocidos por funcionar con Adaptadores genéricos que utilicen el chip CH340: http://www.wch.cn/download/CH341SER_EXE.html <br>
+Tambien puedes encontrar drivers para adaptadores con chip prolific en esta web:
```
Comment (leonelmunozlopez) [inline documentacion/posintegrado/README.md]: * También
* *Chip Prolific*

File: documentacion/posintegrado/README.md [hunk 34]
```diff
@@ -0,0 +1,305 @@
...
+<aside class="notice">
+La comunicación con el POS Integrado se realiza mediante puerto serial RS232.
+</aside>
+
+<aside class="notice">
+Tú eres el responsable de instalar el driver correcto para tu tarjeta o adaptador serial.
+</aside>
+
+<aside class="success">
+Estos drivers son conocidos por funcionar con Adaptadores genéricos que utilicen el chip CH340: http://www.wch.cn/download/CH341SER_EXE.html <br>
```
Comment (leonelmunozlopez) [inline documentacion/posintegrado/README.md]: The link should be HTML code?
<a href="http://www.wch.cn/download/CH341SER_EXE.html" target="_blank">descargar</a>
<br />

File: documentacion/posintegrado/README.md [hunk 35]
```diff
@@ -0,0 +1,305 @@
...
+
+El SDK depende de [libSerialPort](https://sigrok.org/wiki/Libserialport) para la comunicación serial.
+
+Puedes obtener el código desde el repositorio usando git:
+
+```bash
+git clone git://sigrok.org/libserialport
+```
+
+Para compilar en windows necesitaras lo siguiente:
```
Comment (leonelmunozlopez) [inline documentacion/posintegrado/README.md]: necesitarás

File: documentacion/posintegrado/README.md [hunk 36]
```diff
@@ -0,0 +1,305 @@
...
+
+- [msys2 - mingw-w64](http://www.msys2.org/) Puedes descargarlo siguiendo el link y las instrucciones provistas en su sitio web.
+  - Adicionalmente, necesitaras el toolchain para tu arquitectura:
```
Comment (leonelmunozlopez) [inline documentacion/posintegrado/README.md]: necesitarás

File: documentacion/como_empezar/README.md (hunk 30 above)
Comment (afiebig) [reply documentacion/como_empezar/README.md]: sorry the link is broken, can you please elaborate more?

File: documentacion/como_empezar/README.md (hunk 29 above)
Comment (afiebig) [reply documentacion/como_empezar/README.md]: Not related to this PR, it could be a good idea if you submit a PR with spellcheking.

File: documentacion/posintegrado/README.md (hunk 34 above)
Comment (afiebig) [reply documentacion/posintegrado/README.md]: It should be a MD link

File: producto/webpay/README.md [hunk 37]
```diff
@@ -157,5 +157,5 @@ El comercio tiene un plazo de 90 días desde la fecha de venta para anular tran
         </a>
       </div>
     </div>
-  </div>
+  </div>s
```
Comment (leonelmunozlopez) [inline producto/webpay/README.md]: remove "s"

File: producto/posintegrado/README.md [hunk 38]
```diff
@@ -0,0 +1,122 @@
...
+
+Este modelo de administración de llaves es el que se usa para las llaves MAC.
+
+### El modelo DUKPT – Encriptación de PIN
+
+El nuevo método de administración de llaves para PIN que usará Transbank es el denominado ―Llave Única derivada por transacción o DUKPT por sus iniciales en inglés.
+
+Bajo este método los PED son inicializados en un ambiente seguro, con datos de identificación propios de cada PED (Identificador de la llave de derivación, Identificador de PED único y un contador de transacciones iniciado en cero), más una llave inicial que se calcula usando los datos propios de cada PED y la llave de derivación base. Con esta llave inicial se genera la próxima llave de cifrado para PIN. Este proceso se realiza con una función asimétrica (DUKPT del PinPad), es decir, una función de un solo sentido, de forma que el PED no sea capaz de generar ninguna llave anterior a la actual.
+
+### Calculo de MAC
```
Comment (leonelmunozlopez) [inline producto/posintegrado/README.md]: Cálculo

File: referencia/webpay/README.md [hunk 39]
```diff
@@ -205,7 +205,7 @@ débito, en donde quién realiza el pago ingresa al sitio del comercio,
 selecciona productos o servicio, y el ingreso asociado a los datos de la tarjeta
 de crédito o débito lo realiza en forma segura en Webpay.
```
Comment (leonelmunozlopez) [inline referencia/webpay/README.md]: ... de forma segura en Webpay ...
